asv/plugins/summarylist.py

```diff
@@ -23,8 +23,7 @@ def benchmark_param_iter(benchmark):
     if not benchmark['params']:
         yield None, ()
     else:
-        for item in enumerate(itertools.product(*benchmark['params'])):
-            yield item
+        yield from enumerate(itertools.product(*benchmark['params']))
```

Comment:
Hmm, is there a way to leave this? The implicit for loop is tricky here.

Reply:
Do you mean this change breaks the code? Both version should be functionally equivalent; `yield from` is more efficient because the loop is implemented inside Python itself.

Reply:
I've reverted this change and disabled rule [`UP028`](https://docs.astral.sh/ruff/rules/yield-in-for-loop/#yield-in-for-loop-up028).